health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AddingColumns.js

```
@@ -0,0 +1,120 @@
+              title={t("ADD")}
+              onClick={() => {
+                addNewCol(`MP_COL_NAME_${colValues.length+1}`,newColValue)
+                setShowPopUp(false);
+              }}
+            />,
+          ]}
+          sortFooterChildren={true}
+          onClose={() => {
+            setShowPopUp(false);
+          }}
+        ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)